frontends/web/src/routes/account/receive/receive-bb01.tsx

```diff
@@ -201,16 +201,16 @@ export const Receive = ({
                       {t('receive.changeScriptType')}
                     </button>
                   )}
-                  { hasManyScriptTypes && addressDialog && (
-                    <form onSubmit={e => {
-                      e.preventDefault();
-                      setActiveIndex(0);
-                      setAddressType(addressDialog.addressType);
-                      setAddressDialog(undefined);
-                    }}>
-                      <Dialog medium title={t('receive.changeScriptType')} >
-                        {availableScriptTypes.current && availableScriptTypes.current.map((scriptType, i) => (
-                          <div key={scriptType}>
+                  <form onSubmit={e => {
+                    e.preventDefault();
+                    setActiveIndex(0);
+                    setAddressType(addressDialog ? addressDialog?.addressType : addressType);
+                    setAddressDialog(undefined);
+                  }}>
```

Comment:
note to myself: test bb01 receive. There is small change that the form is always rendered now, but shouldn't be a problem.